lua/triptych/init.lua

```diff
@@ -35,15 +35,20 @@ local function toggle_triptych(dir)
   local Diagnostics = config.diagnostic_signs.enabled and diagnostics.new() or nil
   local FileReader = file_reader.new(config.options.syntax_highlighting.debounce_ms)
 
-  local maybe_buf_name = u.cond(dir, {
-    when_true = nil,
-    when_false = vim.api.nvim_buf_get_name(0),
-  })
-  local opening_dir = dir
-    or u.cond(u.is_defined(maybe_buf_name), {
-      when_true = vim.fs.dirname(maybe_buf_name),
-      when_false = vim.fn.getcwd(),
-    })
+  local opening_dir, selected_file = u.eval(function()
+      if dir then
+        -- if dir is given, open it
+        return dir, dir
+      elseif vim.api.nvim_buf_get_option(vim.api.nvim_get_current_buf(), 'buftype') == 'terminal' then
+        -- in case of a terminal buffer, open the current working directory
+        return vim.fn.getcwd(), ''
```

Comment:
```suggestion
        return vim.fn.getcwd(), nil
```

Lua evaluates empty strings as truthy. So this would cause unexpected behaviour [here](https://github.com/simonmclean/triptych.nvim/blob/57d144d9f516fbc118675695fc0f07edf1cbe58b/lua/triptych/view.lua#L272)